nohup: ignoring input
normalizer.cc(50) LOG(INFO) precompiled_charsmap is empty. use identity normalization.
[nltk_data] Downloading package punkt to
[nltk_data]     /home1/w/why16gzl/nltk_data...
[nltk_data]   Package punkt is already up-to-date!
[nltk_data] Downloading package averaged_perceptron_tagger to
[nltk_data]     /home1/w/why16gzl/nltk_data...
[nltk_data]   Package averaged_perceptron_tagger is already up-to-
[nltk_data]       date!
Some weights of the model checkpoint at google/bigbird-roberta-large were not used when initializing BigBirdModel: ['cls.predictions.transform.dense.weight', 'cls.seq_relationship.weight', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.decoder.bias', 'cls.seq_relationship.bias', 'cls.predictions.transform.dense.bias', 'cls.predictions.bias', 'cls.predictions.decoder.weight']
- This IS expected if you are initializing BigBirdModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BigBirdModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
/shared/why16gzl/conda/miniconda38/envs/EMNLP_env/lib/python3.7/site-packages/numpy/lib/function_base.py:393: RuntimeWarning: Mean of empty slice.
  avg = a.mean(axis)
/shared/why16gzl/conda/miniconda38/envs/EMNLP_env/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/shared/why16gzl/conda/miniconda38/envs/EMNLP_env/lib/python3.7/site-packages/sklearn/metrics/_classification.py:1465: UndefinedMetricWarning: F-score is ill-defined and being set to 0.0 due to no true nor predicted samples. Use `zero_division` parameter to control this behavior.
  average, "true nor predicted", 'F-score is', len(true_sum)
date and time = 11/14/2022 15:03:24
Processing MATRES dataset...
  Data processing took: 0:00:32
# of parameters: 364568582

loaded MATRES best model:0511pm-lr5e-6-b20-gpu9942-loss0-dataMATRES-accum1-marker@**@-pair1-acr0-tmarker1-td1-dpn1-mask0.pt
(from epoch 0 )
Running Evaluation on MATRES Test Set...
Eval took: 0:00:22
  macro F1: nan
  micro F1: 0.000
[]
  macro F1: nan
  micro F1: 0.000
Traceback (most recent call last):
  File "demo.py", line 855, in <module>
    scaled_logits = scaled_model(logits)
  File "/shared/why16gzl/conda/miniconda38/envs/EMNLP_env/lib/python3.7/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/mnt/cogcomp-archive/shared/why16gzl/Repositories/SRL_to_TemporalGraph-main/ts.py", line 38, in forward
    return self.temperature_scale(logits)
  File "/mnt/cogcomp-archive/shared/why16gzl/Repositories/SRL_to_TemporalGraph-main/ts.py", line 46, in temperature_scale
    temperature = self.temperature.unsqueeze(1).expand(logits.size(0), logits.size(1))
IndexError: Dimension out of range (expected to be in range of [-1, 0], but got 1)
